python/ql/test/library-tests/ApiGraphs/py3/test_captured_inheritance.py

```diff
@@ -0,0 +1,21 @@
+from foo import A, B
+
+def func():
+    if cond():
+        class Foo(A): pass
+    else:
+        class Foo(B): pass
+
+    class Bar(A): pass
+    class Bar(B): pass
+
+    class Baz(A): pass
+    
+    def other_func():
+        print(Foo)  #$ use=moduleImport("foo").getMember("A").getASubclass() use=moduleImport("foo").getMember("B").getASubclass()
+        print(Bar)  #$ use=moduleImport("foo").getMember("B").getASubclass() MISSING: use=moduleImport("foo").getMember("A").getASubclass()  The MISSING here is documenting correct behaviour
```

Comment:
I disagree. There is no path where the `Bar` printed on this line can be a subclass of `A`.

```suggestion
        print(Bar)  #$ use=moduleImport("foo").getMember("B").getASubclass()
```